.github/pull_request_template.md

```diff
@@ -8,6 +8,9 @@
 
 ## Commentary (optional)
 
+
+<!-- User-facing API changes need the "User-facing API Change" label -->
```

Comment:
non-blocking: You can put this all under one comment block, but it's fine as-is too.